plugin/src/main/kotlin/org/openrewrite/gradle/AndroidProjectSpec.kt

```diff
@@ -0,0 +1,292 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.gradle
+
+import org.intellij.lang.annotations.Language
+import java.io.File
+import java.nio.charset.Charset
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+import java.nio.file.Path
+
+/**
+ * Utility to help with writing android projects to disk to assist with plugin testing
+ */
+class AndroidProjectSpec(
```

Comment:
I'll probably make this a subclass of `GradleProjectSpec` and remove most of this code.

Reply:
This class was not needed and I've deleted it.